lab1/Parser.py

```diff
@@ -0,0 +1,49 @@
+from constants.config import config
+from constants.tasks import tasks
+
+
+class Parser:
+    def __init__(self, args):
+        self.args = args
+        self.commands = []
+
+    def get_params(self):
```

Comment:
probabil, pentru a face un pic mai "curat" codul, ai putea să definești metode pentru fiecare:

```python
if '-u' in self.args:
    self.handle_udp(stuff)
if '-i' in self.args:
    self.handle_intervals(ceva, altceva)
```

and are you really `getting` params here? (the name seems off)

Reply:
altă chestie la care m-am gândit dar nu știu cât e de ok:
ai un dicționar unde cheile sunt: `-u`, `-i`, `-max` and so on, și valorile sunt metodele corespunzătoare. și ai putea sa ai o metodă:

```python
lala_dict = {
'-u': handle_udp,
'-i': handle_intervals
}

def parse_params(self):
    for param in self.args:
        lala_dict[param]()
```
problema aici e cum să identifici care parametri să-i transmiți în metodă